hardhat/deploymentInfo.json

```diff
@@ -30,13 +30,13 @@
       "address": "0xD369D1481B157D1d791F78766660f5ddF715f1f2"
     },
     "ApeRouter": {
-      "address": "0x3D24345bb7FEE3B0B50d66a58946722BBbB26dFf"
+      "address": "0x42c7FfF180Ce8ef5414400696d3BcdD8A6b9354b"
```

Comment:
Hmm, maybe move this change to a separate PR that describes the update to the address...

Reply:
These changes can be thrown away, & should stop happening now that I fixed the issue in setup-hardhat.sh that was causing them

Reply:
Yeah, this isn't a material change. I'll resolve this.